### PR TITLE
Switchlist command

### PR DESCRIPTION
**Pull request by ZFW777**
Update relocate command so that no restart is needed.
Update relocate command so that all task list files can be transferred to new destination.

### CODE REVIEW

**Comment by ZFW777** [comment]
Please don't merge now. One more step to be done.

**Review by Cherlynn — COMMENTED** [review]
Variable/method names make sense
Functions/class/file not too long

File: src/main/java/teamfour/tasc/logic/commands/SwitchlistCommand.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package teamfour.tasc.logic.commands;
+
+import teamfour.tasc.commons.events.storage.RequestTaskListSwitchEvent;
+import teamfour.tasc.commons.core.EventsCenter;
+
+/**
+ * Switches to a new tasklist.
+ */
+public class SwitchlistCommand extends Command {
```
Comment (Cherlynn) [inline src/main/java/teamfour/tasc/logic/commands/SwitchlistCommand.java:9]: Open close principle in Command

File: src/main/java/teamfour/tasc/logic/parser/Parser.java [hunk 2]
```diff
@@ -149,6 +154,21 @@ private Command prepareAdd(String args){
...
+    /**
+     * Parses arguments in the context of the switch list command.
+     *
+     * @param args the file name of list that user wish to switch to. 
+     * @return the prepared command
+     */
+    public Command prepareSwitchlist(String args) {
+        final Matcher matcher = FILE_NAME_ONLY_FORMAT.matcher(args.trim());
+        if (!matcher.matches()) {
+            return new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
```
Comment (Cherlynn) [inline src/main/java/teamfour/tasc/logic/parser/Parser.java:180]: Error handling

File: src/main/java/teamfour/tasc/commons/events/storage/FileRelocateEvent.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package teamfour.tasc.commons.events.storage;
+
+import teamfour.tasc.commons.events.BaseEvent;
+
```
Comment (Cherlynn) [inline src/main/java/teamfour/tasc/commons/events/storage/FileRelocateEvent.java:4]: Maybe put a comment about what this event does

**Comment by yamgent** [comment]
So does this command work yet?

**Review by yamgent — APPROVED** [review]

**Comment by yamgent** [comment]
#116

**Pull request merged**